syntax/nickel.vim

```diff
@@ -8,17 +8,27 @@ syntax region nickelComment start="#" end="$" contains=@Spell oneline display
 
 " Commands
 syntax keyword nickelKeywords
```

Comment:
Nitpick: since there are now multiple groups of keywords maybe this would be clearer as something like `nickelBindings`? Though I guess that would only be true if we _also_ move `rec` down to `nickelMetadata`.

Reply:
Hmm. I agree `nickelKeywords` is a bit vague, but `fun` isn't really a binding keyword though, it's a constructor. It's a bit of a catch-all case, everything that is not metadata nor control flow nor import. But I don't have a great name for it right now.